scripts/Tools/baseline_gen_comp

```diff
@@ -1,265 +0,0 @@
-#!/bin/bash
```

Comment:
1. This can be done with bless_test_results
2, 3: Good point, I don't think that can be trivially done with the scripts in this PR. I'll address this.